glide-core/redis-rs/redis/src/cluster_async/mod.rs

```diff
@@ -1798,9 +1858,8 @@ where
         if !failed_connections.is_empty() {
             Self::refresh_connections(
                 inner,
-                failed_connections,
+                failed_connections.into_iter().collect::<HashSet<String>>(),
```

Comment:
it creates an unnecessary copy. instead you can change failed_connections to be a set to begin with

Reply:
ping